SWLOR.Game.Server/Feature/StatusEffectDefinition/ForceDrainStatusEffectDefinition.cs

```diff
@@ -33,6 +33,11 @@ private void ForceDrain1(StatusEffectBuilder builder)
                 {
                     if (!Ability.GetAbilityResisted(source, target))
                     {
+                        PlaySound("plr_force_absorb");
```

Comment:
You play a sound on this level of the ability but not the others. I think you need to add this to them.

More importantly there's a ton of redundant code here. I'd recommend throwing all of this stuff into a method and have each of them modify the variables.